app/views/dashboard/_subscription_widget.html.erb

```diff
@@ -1,3 +1,5 @@
+<% require 'erb' %>
```

Comment:
Thats normally implied by the .erb extension.  I'm not aware of this being a normal convention, and it doesn't appear in any existing katello or foreman views, so I would recommend removing it.